examples/ngsi_v2/e13_ngsi_v2_secure_fiware_headers.py

```diff
@@ -0,0 +1,111 @@
+        except requests.exceptions.RequestException as err:
+            raise KeycloakPythonException(err.args[0])
+
+    def get_data(self, client_host, headers={}, keycloak_host=None, client_id=None, client_secret=None):
+        """
+        - Get data for a given api.
+        - Mandatory input - Target api, fiware-service and fiware-servicepath headers
+        - Optional Inmput - Keycloak host, Client ID, Client Secret
+        """
+        access_token, expires_in = self.get_access_token(keycloak_host=keycloak_host, client_id=client_id,
+                                                         client_secret=client_secret)
+        headers['Authorization'] = 'Bearer %s' % (access_token)
+        response = requests.get(client_host, headers=headers)
+        return response.text
+
+    def post_data(self, client_host, data, headers={}, keycloak_host=None, client_id=None, client_secret=None):
+        """
+        - Post data for a given api.
+        - Mandatory input - Target api, headers, request body.
+        - Optional Inmput - Keycloak host, Client ID, Client Secret.
+        """
+        access_token, expires_in = self.get_access_token(keycloak_host=keycloak_host, client_id=client_id,
+                                                         client_secret=client_secret)
+        headers['Content-Type'] = 'application/json'
+        headers['Authorization'] = 'Bearer %s' % (access_token)
+        response = requests.post(client_host, data=data, headers=headers)
+        return response
+
+    def patch_data(self, client_host, json, headers={}, keycloak_host=None, client_id=None, client_secret=None):
+        """
+        - Patch data for a given api.
+        - Mandatory input - Target api, headers, request body.
+        - Optional Inmput - Keycloak host, Client ID, Client Secret.
+        """
+        access_token, expires_in = self.get_access_token(keycloak_host=keycloak_host, client_id=client_id,
+                                                         client_secret=client_secret)
+        headers['Content-Type'] = 'application/json'
+        headers['Authorization'] = 'Bearer %s' % (access_token)
+        response = requests.patch(url=client_host, json=json, headers=headers)
+        return response
+
+
+class KeycloakPythonException(Exception):
+    def __init__(self, message):
+        self.message = message
+        super().__init__(self.message)
+
+
+if __name__ == "__main__":
+    # get token from keycloak
+    token, in_sec = KeycloakPython(KEYCLOAK_HOST, CLIENT_ID, CLIENT_SECRET).get_access_token()
+
+    # create secure fiware header with authorisation token
+    fiware_header = FiwareHeaderSecure(service=SERVICE,
+                                       service_path=SERVICE_PATH,
+                                       authorization='Bearer %s' % token)
+
+    # create a context broker client
+    cb_client = ContextBrokerClient(url=CB_URL, fiware_header=fiware_header, session=session)
```

Comment:
@sbanoeon could you add some get/put requests here to show that the request does not need to be adapted. By the way, the token must be consistently queried, right? If so, please also add a hints regarding this